### ISSUE TITLE
Run neural net classifier

### ISSUE THREAD

**Issue by arlyon** [arlyon/dmml#26]
> Make sure you understand and can explain logically the meaning of the confusion matrix, as well as the information contained in the “Detailed Accuracy” field: TP Rate, FP Rate, Precision, Recall, F Measure, ROC Area.

- Run MultilayerPerceptron, experiment with various Neural Network parameters: add or remove nodes, layers and connections, vary the learning rate, epochs and momentum, and validation threshold.
- You will need to work with Weka’s Neural Network Visualiser in order to perform some of the above tasks. You are allowed to use smaller data sets when working with the Visualiser.
- Experiment with relative performance of Neural Networks and changing parameters. Base your comparative study on the output of confusion matrices as well as other metrics (TP Rate, FP Rate, Precision, Recall, F Measure, ROC Area).
